### PR TITLE
[OPS-5793] This dir successfully bakes solr 7 and solr 8 images.

### PR DESCRIPTION
**Pull request by cafuego**
Now with Alpine 3.10.1 and OpenJDK 11 base images.

### CODE REVIEW

**Comment by teodorescuserban** [comment]
Ignoring the PR at @cafuego request. Please ping when review is needed.

**Comment by cafuego** [comment]
Pingelingeling!  Built on all-new Alpine 3.10.1 / Java 11 base images.

File: alpine-solr/7/Dockerfile.tmpl [hunk 1]
```diff
@@ -0,0 +1,68 @@
+FROM unocha/alpine-jdk:%%UPSTREAM%%
+
+MAINTAINER Serban Teodorescu <teodorescu.serban@gmail.com>
```
Comment (teodorescuserban) [inline alpine-solr/7/Dockerfile.tmpl]: maintainer keyword is being deprecated. and i am not the only maintainer. :)

**Review by teodorescuserban — CHANGES_REQUESTED** [review]
other than those two minor changes it looks great!

File: alpine-base-jdk/openjdk/Dockerfile.tmpl [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      org.label-schema.vcs-ref=$VCS_REF \
+      org.label-schema.name="base-php" \
+      org.label-schema.description="This service provides a base Java platform." \
+      org.label-schema.architecture="x86_64" \
+      org.label-schema.distribution="Alpine Linux" \
+      org.label-schema.distribution-version="$UPSTREAM" \
+      info.humanitarianresponse.jre=$VERSION
+
+RUN apk add --update-cache \
+      openjdk${VERSION}-jre && \
```
Comment (teodorescuserban) [inline alpine-base-jdk/openjdk/Dockerfile.tmpl:30]: ugh, use headless jre.

**Review by teodorescuserban — CHANGES_REQUESTED** [review]
hm thats the second change.

File: alpine-base-jdk/openjdk/Dockerfile.tmpl (hunk 2 above)
Comment (cafuego) [reply alpine-base-jdk/openjdk/Dockerfile.tmpl:30]: Yeah, that's what it installs.

**Pull request merged**